content/contributing/documentation/rst_guidelines.rst

```diff
@@ -1,162 +1,1365 @@
+
+   * - .. code-block:: text
+
+          .. cards::
+
+             .. card:: Documentation
+                :target: ../documentation
+                :tag: Step-by-step guide
+                :large:
+
+                Use this guide to acquire the tools and knowledge you need to write documentation.
+
+             .. card:: Content guidelines
+                :target: content_guidelines
+
+                List of guidelines and trips and tricks to make your content shine at its brightest!
+
+             .. card:: RST guidelines
+                :target: rst_guidelines
+
+                List of technical guidelines to observe when writing with reStructuredText.
+
+.. _contributing-rst-document-metadata:
+
+Document metadata
+=================
+
+| Sphinx supports document-wide metadata markups that specify a behavior for the entire page.
+| They must be placed between colons (`:`) at the top of the source file.
+
++-----------------+--------------------------------------------------------------------------------+
+| **Metadata**    | **Purpose**                                                                    |
++-----------------+--------------------------------------------------------------------------------+
+| `show-content`  |  Make a toctree page accessible from the navigation menu.                      |
++-----------------+--------------------------------------------------------------------------------+
+| `show-toc`      |  Show the table of content on a page that has the `show-content` metadata      |
+|                 |  markup.                                                                       |
++-----------------+--------------------------------------------------------------------------------+
+| `code-column`   |  | Show a dynamic side column that can be used to display interactive          |
+|                 |    tutorials or code excerpts.                                                 |
+|                 |  | For example, see                                                            |
+|                 |    :doc:`/applications/finance/accounting/get_started/cheat_sheet`.            |
++-----------------+--------------------------------------------------------------------------------+
+| `hide-page-toc` | Hide the "On this page" sidebar and use full page width for the content.       |
++-----------------+--------------------------------------------------------------------------------+
+| `custom-css`    | Link CSS files (comma-separated) to the document.                              |
++-----------------+--------------------------------------------------------------------------------+
+| `custom-js`     | Link JS files (comma-separated) to the document.                               |
++-----------------+--------------------------------------------------------------------------------+
+| `classes`       | Assign the specified classes to the `<main/>` element of the document.         |
```

Comment:
What are classes? I'm both personally curious and wondering if we should include an explanation.

Reply:
CSS classes are used to apply specific styles to HTML elements but to be fair I don't think we use this. I'll keep it though just in case we need it someday (as per ANV's request :slightly_smiling_face: )
I reorganized the table to list the markups that are the most useful to us first.